ooiservices/app/uframe/plot_tools.py

```diff
@@ -87,7 +88,7 @@ def plot_time_series(self, fig, is_timeseries, ax, x, y, fill=False, title='', x
         if events:
             y = ax.get_ylim()
             for event in events['events']:
-                time = num2date(float(event['start_date'])/1000, units='seconds since 1970-01-01 00:00:00', calendar='gregorian')
+                time = datestr2num(event['start_date']) 
```

Comment:
@Bobfrat and then changed this to give back the correct object for the 'time' container